web/ui/package-lock.json

```diff
@@ -106,7 +106,7 @@
     },
     "mantine-ui": {
       "name": "@prometheus-io/mantine-ui",
-      "version": "0.51.2",
+      "version": "0.49.1",
```

Comment:
This change probably shouldn't be part of the PR?

Reply:
This is automatic I believe
Not sure why its happening but I don't really touch lockfiles

Reply:
Yep, it's from the build process and depends on the underlying Prometheus version it sees, but no need to check it in unless we cut UI releases.